coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -36,3 +36,61 @@ def init_client(cls, crypto_token_path, app_name="CodaV2Client"):
         cred = credentials.Certificate(crypto_token_path)
         app = firebase_admin.initialize_app(cred, name=app_name)
         return cls(firestore.client(app))
+
+    def get_dataset_ids():
+        """
+        Gets unique ids of datasets having included only one id for segmented dataset ids.
```

Comment:
I would try to write these in terms of datasets where you can, with almost a side-note about segments. This is so users don't need to worry about segments when interacting with datasets.

e.g. something like "Gets all the available dataset ids in Coda. For each segmented dataset, returns only the primary dataset id"

Reply:
Thanks updated